### DISCUSSION TITLE
Design update to UI poll

### DISCUSSION THREAD

**Discussion by damongolding** [damongolding/immich-kiosk#96] [Polls]
The proposed change is to the clock and images data background gradient which sit in the bottom left and right corner(s) of these images. Option A is the current design.

![Frame 50](https://github.com/user-attachments/assets/8ed7d757-9c87-4dfe-b4da-e7dafa48abd6)
![Frame 51](https://github.com/user-attachments/assets/6af70c28-31a3-4c5d-8412-2d90a04c048c)

**Comment by github-cli** [comment]
hard gradient, only because the soft one "bleeds" too far into the images

**Reply by damongolding** [reply]
Thanks for the feedback. That is a very valid point, there are no wrong answers.

**Comment by radh21301** [comment]
Would be nice to see the difference when the base image is mostly white. Wonder if the white text will still be visible.

**Comment by damongolding** [comment]
Good shout. Thanks I’ve tested that locally but I can pop up an example.
